buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -132,10 +126,25 @@ class DistributedPerformanceTest extends PerformanceTest {
             doExecuteTests()
         } finally {
             testEventsGenerator.release()
+            generatePerformanceReport()
             cleanTempFiles()
         }
     }
 
+    private void generatePerformanceReport() {
+        FileUtils.cleanDirectory(reportDir)
+        getProject().javaexec(new Action<JavaExecSpec>() {
+            void execute(JavaExecSpec spec) {
+                spec.setMain("org.gradle.performance.results.ReportGenerator")
+                spec.args(resultStoreClass, reportDir.getPath())
+                spec.systemProperties(DistributedPerformanceTest.this.getSystemProperties())
```

Comment:
What system properties do we need to pass here apart from channel and the database properties?